external/glad/OpenGL2.0/include/KHR/khrplatform.h

```diff
@@ -1,3 +1,5 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
these are C sources, shouldn't we use c-comments then? (I know, C99 allows use of //, but ... ?)

Reply:
We use C++14 so our includes can use C99 features. So not worth changing back